packages/formstr-app/src/containers/CreateForm/providers/FormBuilder/index.tsx

```diff
@@ -93,13 +89,24 @@ export default function FormBuilderProvider({
     };
   };
 
+  const deleteDraft = (formTempId: string) => {
+    type Draft = { formSpec: unknown; tempId: string };
+    let draftArr = JSON.parse(
```

Comment:
no need to parse here. localStorage has already parsed data

Reply:
ohhh lemme try

Reply:
UPDATE: getItem signature is `string | null` we need to parse it to be able to iterate over it